reflectivity_ui/interfaces/data_handling/data_set.py

```diff
@@ -291,10 +291,10 @@ def calculate_offspec(self, direct_beam=None):
         for xs in self.cross_sections:
             try:
                 self.cross_sections[xs].offspec(direct_beam=direct_beam)
-            except:
+            except Exception:
```

Comment:
Is it possible to make this `Exception` more specific?

Reply:
I could, but I wasn't changing behavior.